packages/lesswrong/server/repos/UsersRepo.ts

```diff
@@ -543,6 +543,64 @@ class UsersRepo extends AbstractRepo<"Users"> {
     `, [limit])
   }
 
+   async getUsersForInactiveSummaryEmail(limit: number): Promise<(DbUser & {
+     fetchActivitySince: Date,
+   })[]> {
+    return this.getRawDb().manyOrNone(`
+      -- UsersRepo.getUsersForInactiveSummaryEmail
+      SELECT
+        u.*,
+        GREATEST(
+          u."inactiveSummaryEmailSentAt",
+          rs."max_last_updated"
+        ) "fetchActivitySince"
```

Comment:
```suggestion
        GREATEST(
          u."inactiveSummaryEmailSentAt",
          rs."max_last_updated",
          u."createdAt"
        ) "fetchActivitySince"
```

There's one edge case here, which can cause `fetchActivitySince` to be null: if the user hasn't been sent the summary email, and they don't have any read statuses. This can happen with a new user account (you can see in your latest example screenshot that there are some quite old posts included). I think we can address this by just falling back to the account `createdAt` date.